mmv1/products/networkservices/HttpRoute.yaml

```diff
@@ -0,0 +1,507 @@
+    description: |
+      Time the HttpRoute was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    description: Set of label tags associated with the HttpRoute resource.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::Array
+    name: hostnames
+    item_type: Api::Type::String
+    description: |
+          Required. Set of hosts that should match against the HTTP host header to select a HttpRoute to process the request.  
+    required: true
+    send_empty_value: true  
+  - !ruby/object:Api::Type::Array
+    name: meshes
+    item_type: Api::Type::String
+    description: |
+          List of meshes this HttpRoute is attached to, as one of the routing rules to route the requests served by the mesh.  
+    send_empty_value: true 
+  - !ruby/object:Api::Type::Array
+    name: gateways
+    item_type: Api::Type::String
+    description: |
+          List of gateways this HttpRoute is attached to, as one of the routing rules to route the requests served by the gateway.  
+    send_empty_value: true
+  - !ruby/object:Api::Type::Array
+    name: rules
+    description: |
+          Rules that define how traffic is routed and handled.  
+    required: true
+    send_empty_value: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties: 
+        - !ruby/object:Api::Type::Array
+          name: matches
+          description: |
+            A list of matches define conditions used for matching the rule against incoming HTTP requests.
+          send_empty_value: true
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: ignoreCase
+                description: |
+                  Specifies if prefixMatch and fullPathMatch matches are case sensitive.
+                send_empty_value: true
+              - !ruby/object:Api::Type::String
+                name: fullPathMatch
+                description: |
+                  The HTTP request path value should exactly match this value.
+                exactly_one_of:
+                  - fullPathMatch
+                  - prefixMatch
+                  - regexMatch
+              - !ruby/object:Api::Type::String
+                name: prefixMatch
+                description: |
+                  The HTTP request path value must begin with specified prefixMatch.
+                exactly_one_of:
+                  - fullPathMatch
+                  - prefixMatch
+                  - regexMatch
+              - !ruby/object:Api::Type::String
+                name: regexMatch
+                description: |
+                  The HTTP request path value must satisfy the regular expression specified by regexMatch after removing any query parameters and anchor supplied with the original URL.
```

Comment:
Can we add the following to the description?

`For regular expression grammar, please see https://github.com/google/re2/wiki/Syntax`

Reply:
Done